Jenkinsfile

```diff
@@ -55,7 +55,7 @@ pipeline {
         success {
             script{
                 if ( env.BRANCH_NAME == 'master' || env.BRANCH_NAME == 'devel' ) {
-                    slackSend( message: ":rocket: New version for <https://jenkins.einfra.grnet.gr/job/ARGO/job/$PROJECT_DIR/|$PROJECT_DIR>!")
+                    slackSend( message: ":rocket: New version for <https://jenkins.einfra.grnet.gr/job/ARGO/job/$PROJECT_DIR/|$PROJECT_DIR> : $PROJECT_DIR !")
```

Comment:
can we have a different way of printing the URL?
What about other statuses instead of success.

Reply:
We can add the organization as an ENV variable for  example $ENV_ORG. There is also an on failure status bellow

Reply:
I used the BUILD_URL instead of manually building the link. And we have messages for success and failure statuses.